helpdesk_mgmt/security/ir.model.access.csv

```diff
@@ -3,6 +3,7 @@ access_helpdesk_ticket_manager,helpdesk.ticket.manager,model_helpdesk_ticket,gro
 access_helpdesk_ticket_user,helpdesk.ticket.user,model_helpdesk_ticket,group_helpdesk_user,1,1,1,0
 access_helpdesk_ticket_user_personal,helpdesk.ticket.user.personal,model_helpdesk_ticket,group_helpdesk_user_own,1,1,1,0
 access_helpdesk_ticket_portal,helpdesk.ticket.portal,model_helpdesk_ticket,base.group_portal,1,1,0,0
+access_helpdesk_ticket_internal_user,helpdesk.ticket.internal.user,model_helpdesk_ticket,base.group_user,1,0,0,0
```

Comment:
better fix this one by restricting the [smart button](https://github.com/OCA/helpdesk/blob/44b501023dc4b4f06203b72be8fbfc0a16b7e64d/helpdesk_mgmt/views/res_partner_view.xml#L10) to the helpdesk user group

Reply:
Restricted the smart button.
But I could not remove `access_helpdesk_ticket_internal_user` : without it, internal users keeps getting access error

Reply:
that's weird. I'll look into this and come back to you, this *should* work

Reply:
right, this only makes the button invisible. You can restrict the whole view override to the helpdesk user group in order to not have the field on the view at all.

My problem with allowing unrestricted access is that customers have the tendency to post all kinds of sensitive information in tickets. For the other groups we have `ir.rule`s to restrict this, so I could also live with this ACL if you add the internal user group to the rule restricting to own tickets.

Reply:
> You can restrict the whole view override to the helpdesk user group in order to not have the field on the view at all

Right, done, thanks